src/opl.c

```
@@ -1283,6 +1304,38 @@ static char gLastSaveTarget[sizeof(gCustomSettingsPath)];
 static int gHddSettingsFallbackNotice = 0;
 static int gBootHddCommonFallback = 0;
 
+static int checkLoadConfigMMCE(int types)
```

Cppcheck (2.21.0)

[style] The function 'checkLoadConfigMMCE' is never used.

(unusedFunction)